mlapi/job.go

```diff
@@ -99,3 +103,24 @@ func (c *Client) UpdateJob(ctx context.Context, job Job) (Job, error) {
 func (c *Client) DeleteJob(ctx context.Context, id string) error {
 	return c.request(ctx, "DELETE", "/manage/api/v1/jobs/"+id, nil, nil, nil)
 }
+
+// LinkHolidaysToJob links a job to a set of holidays.
+// Only the ID and Holidays fields of the Job struct are used.
+func (c *Client) LinkHolidaysToJob(ctx context.Context, jobID string, holidayIDs []string) (Job, error) {
+	job := Job{
+		ID:       jobID,
```

Comment:
Wait, shouldn't this not be set? I thought the old logic we were clearing it?

Reply:
Whoops, yeah that'd fail, good point!